modules/tinymce/src/core/main/ts/content/PrePostProcess.ts

```diff
@@ -30,21 +53,21 @@ const postProcessGetContent = <T extends GetContentArgs>(editor: Editor, content
   if (args.no_events) {
     return content;
   } else {
-    const eventArgs = Events.fireGetContent(editor, { ...args, content });
-    return eventArgs.content;
+    const outcome = withSerializedContent(content, (c) => Events.fireGetContent(editor, { ...args, content: c }));
+    return outcome.content;
   }
 };
 
 const preProcessSetContent = <T extends SetContentArgs>(editor: Editor, args: T): Result<T, undefined> => {
   if (args.no_events) {
     return Result.value(args);
   } else {
-    const eventArgs = Events.fireBeforeSetContent(editor, args);
-    if (eventArgs.isDefaultPrevented()) {
-      Events.fireSetContent(editor, eventArgs);
+    const outcome = withSerializedContent(args.content, (content) => Events.fireBeforeSetContent(editor, { ...args, content }));
```

Comment:
Is outcome really a better describing name for this it's as abstract as `value` is it to describe that it got changed maybe `processedEventArgs` would be more suitable? But maybe I'm overly picky. So feel free to ignore this if you want to. :smile:

Reply:
Fair point, I'll rename it. I was mainly just trying to show it wasn't the original event args, so `processedEventArgs` would do that.